### DISCUSSION TITLE
Editable ojs chunks?

### DISCUSSION THREAD

**Discussion by lucio-cornejo** [quarto-dev/quarto-cli#4086] [Q&A] [answered]
Would it be possible to be able to modify the code in a ojs chunk from some rendered qmd file?
Kind of like Observable notebooks work.

**Comment by cscheid** [comment, marked as answer]
No, that's not feasible with our compilation model producing plain HTML files that can be served by a regular web server. Observable notebooks are run as an app that requires a custom server.

**Reply by lucio-cornejo** [reply]
Oooh I see.
Still, qmd notebooks with only ojs chunks render fast enough, so, they work great as they are now.
Thanks for the quick reply.
